components/content/QueryState.tsx

```diff
@@ -129,8 +129,5 @@ export const Unavailable: React.FunctionComponent = () => {
 
 // incompleteQueryState checks to see if any of the queries are not yet complete - if so, render a <QueryState/>.
 export const incompleteQueryState = (...results: UseQueryResult[]): boolean => {
-  return results
-    .map(result => [result.isError, result.isLoading, isResultNotFound(result)])
-    .flat()
-    .reduce((accumulator, value) => accumulator || value);
+  return results.some(result => result.isError || result.isLoading);
```

Comment:
1. the check for `result.isError` made the check for `isResultNotFound` redundant, so I removed the latter
2. `results.some` simplifies things